Taskfile.yaml

```diff
@@ -182,7 +182,8 @@ tasks:
       - install-rpk
     vars:
       RPK_LOC: '{{.ARTIFACT_DIR}}/bin/rpk'
-      CA_CRT: '{{ default "ansible/tls/ca/ca.crt" .CA_CRT }}'
+      # this needs to go somewhere else badly
```

Comment:
can you expand upon this? is there an issue open to track that change in the future?

Reply:
Yeah it probably needs an issue. Putting certs in an internal directory was always a bad choice